iphone/cli/commands/_buildModule.js

```diff
@@ -915,12 +915,7 @@ iOSModuleBuilder.prototype.packageModule = function packageModule(next) {
 
 		// 3. platform folder
 		if (fs.existsSync(this.platformDir)) {
-			this.dirWalker(this.platformDir, function (file, name) {
-				var stat = fs.statSync(file);
-				if (name !== 'README.md') {
-					dest.append(fs.createReadStream(file), { name: path.join(moduleFolders, 'platform', path.relative(this.platformDir, file)), mode: stat.mode });
-				}
-			}.bind(this));
+			dest.directory(this.platformDir, path.join(moduleFolders, 'platform'));
```

Comment:
Does this method method ensure that the README.md is not zipped up as well?

Reply:
no, unfortunatly not....
for now I did not find out how to do the symbolic link fix with dirWalker... but is this really an issue?

Reply:
It's a behavioral change on how modules are zipped. I think @cb1kenobi will know more here, as we built the initial script.

Reply:
`dest.directory()` supports a 3rd arg which can be a function: https://github.com/archiverjs/node-archiver/blob/master/lib/core.js#L596-L604. We can use that function to filter out the README.

```suggestion
			dest.directory(
				this.platformDir,
				path.join(moduleFolders, 'platform'),
				entryData => entryData.name === 'README.md' ? false : entryData
			);
```

I can't easily test this code out, so I can't say if it will work. Additionally, it would be helpful if there was a test case.

Interestingly, I see `directory()` will tack the `stats` onto the `entryData` which should include whether the file is a symlink, so it does seem like this is a more robust method.